ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -63,12 +72,17 @@ Rectangle {
 
     function onEnter(event){
         if (event.modifiers === Qt.NoModifier && (event.key === Qt.Key_Enter || event.key === Qt.Key_Return)) {
-            if (emojiSuggestions.visible) {
-                emojiSuggestions.addEmoji();
-                event.accepted = true;
-                return
+            if (txtData.length < messageLimit) {
+                if (emojiSuggestions.visible) {
+                    emojiSuggestions.addEmoji();
+                    event.accepted = true;
+                    return
+                }
```

Comment:
This `if` can be put outside the `if (txtData.length < messageLimit) {` since putting a new emoji should be accepted whether or not the limit is reached

Reply:
Whoops, just fixed this!

Reply:
Whoops, just fixed this!